content/github/importing-your-projects-to-github/importing-source-code-to-github/adding-an-existing-project-to-github-using-the-command-line.md

```diff
@@ -31,27 +31,20 @@ shortTitle: Add a project locally
 1. Initialize the local directory as a Git repository.
 
     ```shell
-    git init -b main
+    git -C <em>path/to/repo</em> init -b main
```

Comment:
Should the first step ("In the command line, navigate to the root directory of your project") be omitted now that we are including `-C <em>path/to/repo</em>`?

Reply:
I agree it's more user-friendly without `-C`. I think @pxrth9 wanted to show how the new `--source` flag works with any path, but we can save that for elsewhere